Conduit/Conduit/bin/Debug/net6.0/appsettings.Development.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
@noor848 again, the bin folder should be in .gitignore

Reply:
donrin the second branch